app/views/upload.scala.html

```diff
@@ -1,6 +1,34 @@
-@()(implicit messages: Messages)
+    @()(implicit messages: Messages)
 
 @main(Messages("upload.header")) {
-    <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
-    <p>Placeholder page for upload</p>
-}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        @progressIndicator(Messages("progress.uploadStep"))
+
+        <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
+        <div class="govuk-form-group">
+            <label class="govuk-label" for="file-selection">
+                @Messages("upload.selectFolder")
+            </label>
+            <input type="hidden" class="file-id-data" name="file-id-data" />
+            <input type="file" id="file-selection" name="files" class="govuk-file-upload" webkitdirectory>
+    
+            <details class="govuk-details" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        @Messages("upload.fileExtensionTitle")
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    @Messages("upload.fileExtensionSummary")
+                </div>
+            </details>
+
+            <button class="govuk-button" type="submit" data-module="govuk-button" role="button">
+                Continue
```

Comment:
Just spotted one more thing that should be in the messages file.

Reply:
I didn't add this to the messages file because on previous pages it is hard coded, so I wasn't sure if I should follow the same pattern or not.

Reply:
Nigel's in the process of moving the other ones, e.g. https://github.com/nationalarchives/tdr-transfer-frontend/blob/adc6d8bc15fa98d13bb0e5743865ac7834b6b1fb/conf/messages#L20

So it would be great if you could move this one too, since it's new.

Reply:
No worries, that's done too now 😄